### PR TITLE
Remove useless blank line

### PR DESCRIPTION
**Pull request by chendotjs**
Remove useless blank line, make it more conformant.

### CODE REVIEW

**Comment by chendotjs** [comment]
/cc @tengqm 
ptal, thanks  :)

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -161,7 +161,6 @@ kubectl get services --sort-by=.metadata.name
 kubectl get pods --sort-by='.status.containerStatuses[0].restartCount'
 
 # List PersistentVolumes in test namespace sorted by capacity
-
```
Comment (bradtopol) [inline content/en/docs/reference/kubectl/cheatsheet.md]: Great catch

**Pull request merged**